packages/remix-route-convention/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@remix-run/route-convention",
```

Comment:
Maybe `@remix-run/folder-route-convention` or `@remix-run/nested-folder-route-convention` for clarity?